Makefile.include

```diff
@@ -1,6 +1,10 @@
 # 'Makefile.include' directory, must be evaluated before other 'include'
 _riotbase := $(dir $(lastword $(MAKEFILE_LIST)))
 
+# include RIOT_MAKEFILES_GLOBAL_PRE configuration files
+# allows setting user specific system wide configuration parsed before
```

Comment:
What means "before" and "after"?

Reply:
before and after the rest of the Makefile.include file content, whatever that means in practice.
I can precise with this or I welcome suggestions.

I do not really know if it should be described as low level "parsed files at the top and the bottom of Makefile.include" or more, "conceptually allows wrapping Makefile.include with added functionalities that should be defined before other ones or can depend on defined variables from Makefile.include".

I see it as low level developer oriented feature, and for me, the "parsed files" is more clear than the abstract one, even if it does not tell me what it can be used for.
The what it could be used for, could come more in a dedicated documentation I would think with examples.